easybuild/easyconfigs/a/Annif/Annif-0.40.0-foss-2019a-Python-3.7.2.eb

```diff
@@ -0,0 +1,108 @@
+easyblock = 'PythonBundle'
+
+name = 'Annif'
+version = '0.40.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://github.com/NatLibFi/Annif'
+description = "Annif is a multi-algorithm automated subject indexing tool for libraries, archives and museums."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+
+dependencies = [
+    ('Python', '3.7.2'),
+    ('PyYAML', '5.1'),
+    ('SciPy-bundle', '2019.03'),  # required for numpy/scipy
+    ('scikit-learn', '0.20.3'),
+]
+
+use_pip = True
+
+exts_default_options = {
+    'source_urls': [PYPI_SOURCE],
+    'source_tmpl': SOURCE_TAR_GZ,
+}
+
+exts_list = [
+    ('sklearn', '0.0', {
```

Comment:
Should this really included?

Reply:
@boegel Any update on @Micket's question?

Reply:
@Micket It's required to avoid that the stub `sklearn` package is auto-downloaded as a dependency...